lib/paynet_api/client.rb

```diff
@@ -23,6 +23,7 @@ def search_for_company(params)
       def get request
         connection = Faraday.new request.url do |conn|
           conn.adapter Faraday.default_adapter
+          conn.proxy "http://quotaguard5236:5cb872578a51@us-east-static-01.quotaguard.com:9293"
```

Comment:
I'm glad this is so easy but let's not hard code it.. we need to pass it to the library as part of it's configuration

Reply:
@drewnichols do you mean define it in the paynet api base response class?